RELEASE_NOTES.rst

```diff
@@ -21,6 +21,63 @@
 
 .. towncrier release notes start
 
+Airflow 2.10.5 (2025-01-28)
+---------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+- Ensure teardown tasks are executed when DAG run is set to failed
```

Comment:
This section should be formatted. Look at previous releases

Reply:
Reformatted it, PTAL.

Reply:
Reformatted it, PTAL.

Reply:
Reformatted it, PTAL.